src/plugins/data/common/index_patterns/index_patterns/types.ts

```diff
@@ -33,3 +33,16 @@ export interface TypeMeta {
   aggs?: Record<string, AggregationRestrictions>;
   [key: string]: any;
 }
+
+export interface IndexPatternSpec {
+  _id: string;
+  _type: string;
```

Comment:
whats a type ? are this saved object properties ?

Reply:
yes, its a saved object property.